shell/public/index.html

```diff
@@ -7,13 +7,13 @@
             content="width=device-width, initial-scale=1, shrink-to-fit=no"
         />
         <meta name="theme-color" content="#ffffff" />
-        <!--
-          manifest.json provides metadata used when your web app is installed on a
-          user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-          <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+        <!-- 
+            manifest.json provides metadata used when your web app is installed on a
+            user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
         -->
+        <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```

Comment:
PWA Manifest added to `index.html`; the manifest is generated on both `yarn start` and `yarn build` commands